packages/article/src/Domain/Model/ArticleDimensionContent.php

```diff
@@ -97,4 +103,12 @@ public static function getResourceKey(): string
     {
         return ArticleInterface::RESOURCE_KEY;
     }
+
+    public static function getEffectiveDimensionAttributes(array $dimensionAttributes): array
```

Comment:
I'm not really happy about this part, but I'm not sure how we could make it more extensible 🤔